Making for ST-Link firmware size

`I've` tried running:
make PROBE_HOST=stlink ST_BOOTLOADER=1
As suggested in the readme, but i get this output:

 LD      blackmagic.elf
Memory region         Used Size  Region Size  %age Used
             rom:      139608 B       128 KB    106.51%
             ram:       11704 B        20 KB     57.15%
/usr/lib/gcc/arm-none-eabi/9.2.1/../../../arm-none-eabi/bin/ld: address 0x8021fc0 of blackmagic.elf section `.text' is not within region `rom'
/usr/lib/gcc/arm-none-eabi/9.2.1/../../../arm-none-eabi/bin/ld: blackmagic.elf section `.ARM.exidx' will not fit in region `rom'
/usr/lib/gcc/arm-none-eabi/9.2.1/../../../arm-none-eabi/bin/ld: address 0x8021fc0 of blackmagic.elf section `.text' is not within region `rom'
/usr/lib/gcc/arm-none-eabi/9.2.1/../../../arm-none-eabi/bin/ld: region `rom' overflowed by 8536 bytes
collect2: error: ld returned 1 exit status
make[1]: *** [Makefile:125: blackmagic.elf] Error 1
make: *** [Makefile:17: all] Error 2

The branch I've used is "main", 3bc59eb7e6adad89d9ab724f6eaefda1803bd115.

--- a/src/command.c
+++ b/src/command.c
@@ -141,7 +141,9 @@ int command_process(target_s *t, char *cmd)
 
 	/* Tokenize cmd to find argv */
 	argc = 0;
-	for (const char *part = strtok(cmd, " \t"); part; part = strtok(NULL, " \t"))
+	/* Reentrant strtok needs a state pointer to the unprocessed part */
+	char *token_state = NULL;
+	for (const char *part = strtok_r(cmd, " \t", &token_state); part; part = strtok_r(NULL, " \t", &token_state))
 		argv[argc++] = part;
 
 	/* Look for match and call handler */

